packages/core-ethereum/crates/core-ethereum-db/src/db.rs

```diff
@@ -263,6 +237,15 @@ impl<T: AsyncKVStorage<Key = Box<[u8]>, Value = Box<[u8]>>> HoprCoreEthereumDbAc
             .collect::<Vec<UnacknowledgedTicket>>())
     }
 
+    async fn update_acknowledged_ticket(&mut self, ticket: &AcknowledgedTicket) -> Result<()> {
```

Comment:
Could there instead be `upsert_acknowledged_ticket`?

Reply:
We cannot insert at this point, only update.